src/services/telemetryService.js

```diff
@@ -16,8 +16,14 @@ export class TelemetryService {
       .map(toRulesModel);
   }
 
-  /** Returns a list of alarms */
+  /** Returns a list of alarms (all statuses) */
   static getAlarms(params = {}) {
+    return HttpClient.get(`${ENDPOINT}alarms?${stringify(params)}`)
+      .map(toAlarmsModel)
```

Comment:
Looking at the response objects it doesn't seem like `toAlarmsModel` can be resused here.

Reply:
The only thing missing is the count.  But I can make a new one